### DISCUSSION TITLE
Creating a test cluster?

### DISCUSSION THREAD

**Discussion by Just-Insane** [onedr0p/cluster-template#773] [Q&A] [answered]
Hello,

I am hoping to set up a test cluster that can be used to test changes (and new) apps before they are deployed to production.

Looking at the flux docs on repo layouts, it seems like the easiest method would be to set up a second repository that’s configured against my test cluster. Trying to use a monorepo approach seems like it'd require restructuring the whole repo which isn't ideal.

Has anyone else gone this route? Do you have any tips on how to set this up and operate it in practice?

**Comment by onedr0p** [comment, marked as answer]
It's possible it just requires paths to change and resources to be updated.

@bjw-s does something like this.

https://github.com/bjw-s/home-ops/tree/main/kubernetes
